### DISCUSSION TITLE
No audio in recorded videos

### DISCUSSION THREAD

**Discussion by claysam75** [photobooth-app/photobooth-app#287] [Q&A]
I've just set this up on my windows machine to test before installing on a pi for my wedding. I've got it successfully capturing video (albeit at very low frame rate) from my USB webcam, however there is no sound with the video. Is this expected behaviour?

**Comment by mgineer85** [comment]
Yes, no audio currently supported

**Reply by hendikawp** [reply]
I need this feature to

**Comment by WebfinityDesign** [comment]
Will it be in the future? Would be nice to use as a video guestbook (like the audio guestbook, but with video)

**Reply by mgineer85** [reply]
If somebody implements this in a platform independent way, it could be supported.
It’s not on my list currently, but feel free to send a feature request or even better a pull request 🙏
